src/System.Private.ServiceModel/tests/Benchmarks/PerfUsingCoreWCFService/CoreWCFPerfService/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using CoreWCF;
+using CoreWCF.Configuration;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+
+namespace WCFCorePerfService
+{
+    public class Program
+    {        
+        static void Main(string[] args)
+        {
+            string filePath = Path.Combine(Environment.CurrentDirectory, "WCFCorePerfService.exe");
+            Console.WriteLine(filePath);
```

Comment:
I think a better option is `Assembly.GetEntryAssembly().Location` as the exe doesn't need to be launched from the current directory.

Reply:
Assembly.GetEntryAssembly().Location will return the full path of coreWCFPerfService.dll.  but I need to get the full path of  exe and add into firewall